backend/tournesol/utils/constants.py

```diff
@@ -18,4 +18,5 @@
 COMPARISON_MAX = 10.0
 
 # Default weight for a criteria in the recommendations
+# FIXME: the default weight used by the front end is 50, not 10
```

Comment:
They don't necessarily need to be the same. The backend can assume 10 when nothing is specified & the frontend can initialize a user's search by default with a value of 50.

I vaguely remember discussing with @amatissart to make it 0 by default. I don't remember well what makes a criteria included or not in the scores used for ranking.